content/github/collaborating-with-issues-and-pull-requests/syncing-a-fork.md

```diff
@@ -20,16 +20,16 @@ Before you can sync your fork with an upstream repository, you must [configure a
   > remote: Total 62 (delta 27), reused 44 (delta 9)
   > Unpacking objects: 100% (62/62), done.
   > From https://{% data variables.command_line.codeblock %}/<em>ORIGINAL_OWNER</em>/<em>ORIGINAL_REPOSITORY</em>
-  >  * [new branch]      master     -> upstream/master
+  >  * [new branch]      main     -> upstream/main
   ```
-4. Check out your fork's local `master` branch.
+4. Check out your fork's local `main` branch.
   ```shell
-  $ git checkout master
-  > Switched to branch 'master'
+  $ git checkout main
+  > Switched to branch 'https://github.com/todbot/qtpy-tricks.git'
```

Comment:
```suggestion
  > Switched to the default branch
```

Let's make this comment a little more maintainable 😄  and relevant for everyone using GitHub. We like to make it general like this so that everyone can understand the information in a variety of contexts.

Reply:
```suggestion
  > Switched to branch 'main'
```

This is CLI output directly from `git`, and reflects whatever was written in the `git checkout BRANCH` line just above. Since it was `git checkout main`, "Switched to branch `main`" is the expected output.

Reply:
I didn't realize it was output! I thought it was some kind of commented out something 😄  thank you for the clarification ❤️